nltk/draw/table.py

```diff
@@ -139,7 +134,7 @@ def __init__(self, master, columns, column_weights=None, cnf={}, **kw):
             # the default listbox behavior, which scrolls):
             lb.bind("<B1-Leave>", lambda e: "break")
             # Columns can be resized by dragging them:
-            l.bind("<Button-1>", self._resize_column)
+            lb.bind("<Button-1>", self._resize_column)
```

Comment:
This was a breaking bug that prevented `MultiListBox` from being instantiated. It has been in here for years. I suppose nobody uses it?

Reply:
Wow... yes, this is telling, isn't it